pineappl/src/grid.rs

```diff
@@ -1389,10 +1389,22 @@ impl Grid {
     }
 
     fn symmetrize_channels(&mut self) {
-        if self.key_values().map_or(false, |map| {
-            map["initial_state_1"] != map["initial_state_2"]
-        }) {
-            return;
+        let map = self.key_values();
+        if map.is_some() {
+            let map = map.unwrap();
```

Comment:
Merge this line and the one above into one containing:
```rust
if let Some(map) = map {
    // ...
}
```
This is shorter and you avoid using [`.unwrap()`](https://github.com/NNPDF/pineappl/blob/master/CONTRIBUTING.md#coding-guidelines).

Reply:
done